lib/widgets/add_todo.dart

```diff
@@ -16,16 +20,29 @@ class AddTodo extends StatefulWidget {
 
 class _AddTodoState extends State<AddTodo> {
   final _todoController = TextEditingController();
+  final _editedTodo = Todo(
```

Comment:
Where do you use this?

Reply:
I am thinking if I need it when editing an existing todo.

Reply:
well, you are not passing anything when calling `AddTodo` screen after clicking the pencil (edit) icon. That is something to think about, yes?